lib/cmr/cmr_client.rb

```diff
@@ -99,10 +99,18 @@ def test_query(query, token, headers = {})
     end
 
     def add_collection_assocations_to_service(concept_id, collection_ids, token)
+      add_collection_assocations(concept_id, collection_ids, token, 'services')
+    end
+
+    def add_collection_assocations_to_tool(concept_id, collection_ids, token)
+      add_collection_assocations(concept_id, collection_ids, token, 'tools')
+    end
+
+    def add_collection_assocations(concept_id, collection_ids, token, concept_type)
```

Comment:
You might consider adding something like `concept_type = (concept_id[0] == 'T' ? 'tools' : 'services')` instead of using different methods. The way I see it, the `add_collection_associations` method is only used for non-collection metadata so maintenance would be low if we just skimmed the first character of the concept-id. Your methods are a bit more readable though, so either way works

Reply:
Resolving this by changing the call in the controller.